README.md

```diff
@@ -19,6 +19,36 @@ Run the services locally with Docker Compose
 $ docker-compose up
 ```
 
+### Set up AWS SQS queue locally
+
+Run AWS services locally using localstack.
+
+```bash
+$ docker run --rm -p 4566:4566 -p 4571:4571 localstack/localstack
+```
+
+To create an SQS queue on the localstack instance.
+```bash
+aws \
+sqs create-queue \
+--queue-name local-queue \
+--endpoint-url http://localhost:4566 \
+--region eu-west-2 \
```

Comment:
```suggestion
--region eu-west-2
```
I had to remove the `\` to get this to run

Reply:
you could make it easier and just write `aws sqs create-queue --queue-name local-queue --endpoint-url http://localhost:4566 --region eu-west-2`